docs/arch/adr-001-send-unpublish-emails-for-single-pages.md

```diff
@@ -0,0 +1,61 @@
+# Decision record: reinstate unpublish emails for single page subscriptions
+
+Date: 2021-12-03
+
+## Context
+
+We have just launched a new email notifications feature that allows users to subscribe to updates on a single page.
+Currently this feature is only enabled on a few selected pages (eg. [Open standards for government][open-standards-for-government]),
+but it is expected to roll out to the majority of pages and content types by early 2022.
+There is a user need to let subscribers know when a page they are subscribed to has been withdrawn or redirected.
```

Comment:
Minor: it'd be better to call out what the user need is than to state there is one - otherwise it's somewhat left as an exercise of the reader to define that need.

Reply:
Will amend

Reply:
Have extended this cautiously. I'm wary about the user need direclty as no-one has asked for this feature. However we believe that given the demonstrated need for users who want to stay up to date, this fits as a logical scenario it'd be unexpected if we didn't handle.